src/rpc/blockchain.cpp

```diff
@@ -1512,6 +1512,9 @@ UniValue getchaintxstats(const JSONRPCRequest& request)
         }
     }
 
+    if (pindex->nHeight < 2) {
+        throw JSONRPCError(RPC_INVALID_PARAMETER, "Block's height should be 2 or more.");
```

Comment:
IMO the error should be `RPC_MISC_ERROR` since it's not a parameter. The message should be something like `Chain height should be greater than 1`.